lucene/core/src/java/org/apache/lucene/index/DocValuesUpdate.java

```diff
@@ -152,12 +152,12 @@ static BytesRef readFrom(DataInput in, BytesRef scratch) throws IOException {
     }
 
     NumericDocValuesUpdate(Term term, String field, Long value) {
-      this(term, field, value != null ? value.longValue() : -1, BufferedUpdates.MAX_INT, value != null);
+      this(term, field, value != null ? value : -1, BufferedUpdates.MAX_INT, value != null);
     }
 
 
-    private NumericDocValuesUpdate(Term term, String field, long value, int docIDUpTo, boolean hasValue) {
```

Comment:
previous version was correct camel case (upTo).

Reply:
There were 16 instances of `Upto` and 4 of `UpTo` so I went with the more common one for consistency. Happy to switch the other way if it's more correct according to English. Looking it up now and looks like "upto" isn't a word?